metadata/metadata.go

```diff
@@ -94,6 +94,7 @@ const (
 	StatusCode          = "status code"
 	Syscall             = "system calls"
 	Thread              = "threads"
+	Timestamp           = "timestamp"
```

Comment:
I think leaving as a timestamp is correct because:

1. bosun has epoch and since functions that work with timestamps
2. the thing being measured here is the expiration date of a ssl certificate, and i think the thing we want is to record exactly what's in the certificate. that is, we can record the facts, and compute what we want from that.